app/controllers/v1/sessions_controller.rb

```diff
@@ -171,34 +158,27 @@ def user_logout(saml_response)
       end
     end
 
-    def login_stats_success(saml_response, user_session_form = nil)
-      tracker = url_service(user_session_form&.saml_uuid).tracker
-      tags = ["context:#{saml_response.authn_context}", VERSION_TAG]
-      StatsD.increment(STATSD_LOGIN_NEW_USER_KEY, tags: [VERSION_TAG]) if request_type == 'signup'
-      # track users who have a shared sso cookie
-      StatsD.increment(STATSD_LOGIN_SHARED_COOKIE, tags: tags)
-      StatsD.increment(STATSD_LOGIN_STATUS, tags: tags + ['status:success'])
-      StatsD.measure(STATSD_LOGIN_LATENCY, tracker.age, tags: tags)
-      callback_stats(:success, saml_response)
-    end
-
     def new_stats(type)
       tags = ["context:#{type}", VERSION_TAG]
       StatsD.increment(STATSD_SSO_NEW_KEY, tags: tags)
       StatsD.increment(STATSD_SSO_NEW_FORCEAUTH, tags: tags) if force_authn?
       StatsD.increment(STATSD_SSO_NEW_INBOUND, tags: tags) if inbound_ssoe?
     end
 
-    def login_stats(status, saml_response, user_session_form = nil)
+    def login_stats(status, _saml_response, user_session_form)
+      tracker = url_service(user_session_form&.saml_uuid).tracker
+      type = tracker.payload_attr(:type)
       case status
       when :success
-        login_stats_success(saml_response, user_session_form)
+        tags = ["context:#{type}", VERSION_TAG]
```

Comment:
`tags` here and in the failure case have thee same value, these could move out of the case and be defined after `type`